modules/apps/foundation/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/liferay/init.es.js

```diff
@@ -37,7 +37,7 @@ var initSPA = function(callback) {
 					}
 
 					var excluded = Liferay.SPA.excludedPaths.some(
-						(excludedPath) => url.indexOf(excludedPath) === 0
+						(excludedPath) => url.indexOf(themeDisplay.getPathContext() + excludedPath) === 0
```

Comment:
Hey guys, I'm not sure about this change. Since the excluded paths are a global config, users should be able to exclude any path, not only the ones starting with `themeDisplay.getPathContext()`. Also, this would potentially be a breaking change for some users.